test/common.js

```diff
@@ -423,8 +423,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
Done. I tried running the test under v4 just to see it, but there are other bits of modern elsewhere in common.js :/